charts/kube-prometheus-stack/hack/sync_prometheus_rules.py

```diff
@@ -28,82 +26,107 @@ def new_representer(dumper, data):
 
 
 # Source files list
-#
-# Comment the alerting rules which have been migrated to Whizard Alert Extension
 charts = [
     # {
-    #     'source': 'https://raw.githubusercontent.com/WhizardTelemetry/kse-prometheus/kse/manifests/alertmanager/alertmanager-alertRuleGroups.yaml',
+    #     'source': 'https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/main/manifests/alertmanager-prometheusRule.yaml',
```

Comment:
I'm thinking do we need to maintain 2 versions of sync script 'sync_prometheus_rules.py', one in this repo, another in whizardAlerting?